libs/dh/metering-point/feature-overview/src/graphql/get-metering-point-by-id.graphql

```diff
@@ -33,6 +33,7 @@ query GetMeteringPointById(
     meteringPointId
     isEnergySupplier(energySupplierActorGln: $actorGln)
     isGridAccessProvider(gridAccessProviderActorGln: $actorGln)
+    electricalHeatingStartDate
```

Comment:
suggestion: let's add it to the mock data to get rid of a console error

![Uploading image.png…]()